api/v1alpha3/condition_consts.go

```diff
@@ -47,6 +47,15 @@ const (
 	// while installing the container storage interface  addon; those kind of errors are usually transient
 	// the operation is automatically re-tried by the controller.
 	CSIProvisioningFailedReason = "CSIProvisioningFailed"
+
+	// VCenterAvailableCondition documents the connectivity with vcenter
+	// for a given VSphereCluster
+	VCenterAvailableCondition clusterv1.ConditionType = "VCenterAvailable"
+
+	// VCenterUnreachableReason (Severity=Warning) documents a VSphereCluster controller detecting
+	// issues with VCenter reachability while provisioning the load balancer;
+	// those kind of errors are usually transient and failed provisioning are automatically re-tried by the controller.
+	VCenterUnreachableReason = "VCenterUnreachable"
```

Comment:
Any thoughts on if this should be an **Error** instead of a **Warning**, since this seems to be an absolute blocker for the normal functioning of CAPV.